app/javascript/controllers/index.js

```diff
@@ -4,11 +4,83 @@
 
 import { application } from "./application"
 
-import ColorThemeController from "./color_theme_controller"
-application.register("color-theme", ColorThemeController)
+import RubyUi__AccordionController from "./ruby_ui/accordion_controller"
```

Comment:
I was wondering if it would be better to have a ruby_ui.js file to add all those imports and registering and here on index.js we would have only the import for ruby_ui.js That way ruby_ui.js will be a file that is meant to be modified by the generators only.

Reply:
@cirdes this file isn't meant to be edited manually, it's generated/updated it by running `rails g stimulus xxx` or `rake stimulus:manifest:update`. 

So, to keep simplicity and rails default behavior, our generators just add the controller to `app/javascript/controllers/ruby_ui/xyz_controller.js` and  run `rake stimulus:manifest:update` (in case of esbuild). For importmap apps updating manifest (this index.js file) isn't necessary because `eagerLoadControllersFrom("controllers", application)` loads and registers all controllers inside `app/javascript/controllers` folder.

Reply:
@stephannv , that makes sense. I have the bad habit of editing the index.js. but the correct way is to use `rails g stimulus` like you said!